core/domain/story_domain.py

```diff
@@ -925,8 +927,9 @@ def is_node_upcoming(self) -> bool:
         """
         current_time_msecs = utils.get_current_time_in_millisecs()
         planned_publication_date_msecs = (
-            utils.get_time_in_millisecs(self.planned_publication_date) if
-            self.planned_publication_date else None)
+            self.planned_publication_date_msecs if
+            self.planned_publication_date_msecs else None
```

Comment:
Ditto: re "if X else None" condition. Is it necessary?

Reply:
Thank you for pointing this out. I apologize for the oversight on my part.